pages/models.py

```diff
@@ -2179,12 +2179,13 @@ def tutor_books(self):
         books = Book.objects.filter(tutor_marketing_book=True).order_by('path')
         book_data = []
         for book in books:
-            book_data.append({
-                'id': book.id,
-                'slug': 'books/{}'.format(book.slug),
-                'title': book.title,
-                'cover_url': book.cover_url,
-            })
+            if book.book_state is not 'Retired' and book.book_state is not 'Draft':
```

Comment:
I think you can shorten this to book.book_state not in ['Retired', 'Draft']

Reply:
Done